README.md

```diff
@@ -18,13 +18,11 @@ Proxy-Wasm extensions across different proxies.
 * [C++ SDK]
 * [Go (TinyGo) SDK]
 * [Rust SDK]
-* [Zig SDK]
```

Comment:
Hi @mathetake ,
I'm sorry that you want to remove mention of the Zig SDK, but as the author of it, that is your prerogative. Would you be open to retaining a historical link to it from Proxy-Wasm documentation if it made clear that the SDK is archived and no longer being updated?

Reply:
+1 to leaving it mentioned as "unmaintained" until, hopefully, another version emerges.

Reply:
sorry, I am planning to delete the repo after communicating to my friend who is using it, not being archived, so it would be just a dead link.